### PR TITLE
SSCS-3236 Job scheduler JAR

### PR DESCRIPTION
**Pull request by jackmaloney**
- Add bintray config to job scheduler to allow it to be exported to a JAR

### CODE REVIEW

File: Jenkinsfile_parameterized [hunk 1]
```diff
@@ -4,7 +4,7 @@
 properties([
 		parameters([
 			string(name: 'PRODUCT_NAME', defaultValue: 'rpe', description: ''),
-			string(name: 'APP', defaultValue: 'job-scheduler', description: ''),
+			string(name: 'APP', defaultValue: 'sscs-job-scheduler', description: ''),
```
Comment (timja-kainos) [inline Jenkinsfile_parameterized]: you should change the product name not the app name

File: infrastructure/variables.tf [hunk 2]
```diff
@@ -5,7 +5,7 @@ variable "product" {
 
 variable "microservice" {
   type    = "string"
-  default = "job-scheduler"
+  default = "sscs-job-scheduler"
```
Comment (timja-kainos) [inline infrastructure/variables.tf]: the microservice name hasn't changed just the product name

Comment (jackmaloney) [reply infrastructure/variables.tf]: It's going to be a new micro service though, Job-Scheduler will be a different app

Comment (jackmaloney) [reply infrastructure/variables.tf]: Updating now to match our sscs product names

**Pull request merged**